File: NotesAppJv/app/src/main/java/com/henriquesbraga/notesappjv/activity/main/MainView.java
package com.henriquesbraga.notesappjv.activity.main;

import com.henriquesbraga.notesappjv.model.Note;

import java.util.List;

public interface MainView {
    void showLoading();
    void hideLoading();
    void onGetResult(List<Note> notes);
    void onErrorLoading(String message);
}

File: NotesAppKt/app/src/main/java/com/henriquesbraga/notesappkt/activity/main/MainView.kt
package com.henriquesbraga.notesappkt.activity.main

import com.henriquesbraga.notesappkt.model.Note

interface MainView {
    fun showLoading();
    fun hideLoading();
    fun onGetResult(notes: ArrayList<Note>);
    fun onErrorLoading(message: String);
}